bazel/external/boringssl.patch

```diff
@@ -0,0 +1,11 @@
+--- src/tool/server.cc	2023-01-20 13:31:06.619119943 -0800
++++ src/tool/server.cc	2023-01-20 13:30:59.707052958 -0800
+@@ -206,7 +206,7 @@
+   BIO_puts(bio.get(), "HTTP/1.0 200 OK\r\nContent-Type: text/plain\r\n\r\n");
+   PrintConnectionInfo(bio.get(), ssl);
+ 
+-  char request[4];
++  char request[74];
+   size_t request_len = 0;
+   while (request_len < sizeof(request)) {
+     int ssl_ret =
```

Comment:
Unfortunately `bssl` is not fully featured enough as a http server to work for our test case. Since it only reads 4 bytes of data, stirling never sees enough data to identify its an HTTP request.

I attempted to use nginx linked with BoringSSL as a test case, but ran into complications with getting it to build. Other potential options include using a c++ natsc client (since we link [BoringSSL](https://github.com/pixie-io/pixie/blob/68fc3cf8734a4a3359b2aa20cd07147659dddee8/third_party/BUILD.bazel#L70)), writing another BoringSSL C++ application or using something upstream. Unfortunately our upstream options are limited since we don't support tracing non BIO native applications (otherwise Envoy would work).

This patch is minor enough that I believe this is an acceptable tradeoff. The other options mentioned above require maintaining another test client or server application, which I wanted to avoid. I would rather invest that time in building new functionality that overlaps with the needs of this test case (a Clickhouse protocol parser would allow us to use a clickhouse server).